syntax = "proto3"

service CountService {
	rpc Count(CountRequest) returns (CountResponse); 
}

message CountRequest {
	string query = 1;
}

message CountResponse {
	int count = 1;
}
